CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Syberia
+
+Want to show Syberia some love? Help out by contributing!
+
+ - [Question?](#question)
+ - [Bug?](#issue)
+ - [Feature Request?](#feature)
+ - [Submission Guidelines](#submit)
+
+
+## <a name="question"></a> Got a Question?
+
+If you have questions about how to use Syberia, please direct these to the [Google Group](http://www.example.com)
+discussion list or [StackOverflow](http://stackoverflow.com/questions/tagged/syberia). We are also available on [Gitter](http://www.example.com).
+
+
+## <a name="issue"></a> Found a Bug?
+If you find a bug in the source code or a mistake in the documentation, please [submit an issue](https://github.com/syberia/syberia/issues) to
+our [GitHub Repository][https://github.com/syberia/syberia]. Even better you can submit a Pull Request with a fix! :)
+
+But please see [Submission Guidelines](#submit) before filing an issue.
+
+
+## <a name="feature"></a> Want a Feature?
+If you find a bug in the source code or a mistake in the documentation, please [submit an issue](https://github.com/syberia/syberia/issues) to
+our [GitHub Repository][https://github.com/syberia/syberia].
+
+But please see [Submission Guidelines](#submit) before filing an issue.
+
+
+## <a name="submit"></a> Submission Guidelines
+
+### Submitting an Issue
+Before you submit your issue search the archive, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Ideally, an issue will contain:
+
+* A stacktrace of the issue (Use [bettertrace](github.com/robertzk/bettertrace) to capture stacktraces more effortlessly)
```

Comment:
I think I've decided that bettertrace will be [bundled](https://github.com/syberia/syberia/blob/release_candidate1/R/zzz.R#L7).